manuscript/the-meat.txt

```diff
@@ -461,14 +461,14 @@ We start with the year fragment:
 
 We get the list of years in the current data and return either an empty string or `in Year`. If there’s only one year in the data, we assume it’s been filtered by year.
 
-The render function is going to be simple as well:
+The `render` function is going to be simple as well:
 
 {crop-start-line=110,crop-end-line=129,linenos=off,lang=jsx}
 <<[Title.render function](code_samples/meta.jsx)
 
 We’re returning an `<h2>` element with a title that includes some dynamic tags. We change `made` to `make` based on whether or not we’re adding `in years`, and we used `d3.mean` to get the average.
 
-Now we can just add the title into H1BGraph’s `render` method. You can use as many as you want, wherever you want, but I think putting it above the graph works best.
+Now we can just add the title into H1BGraph’s render method. You can use as many as you want, wherever you want, but I think putting it above the graph works best.
```

Comment:
Yep. I'm going to run the text through Marked 2 in the future with a highlight for `just` and `simply` and the like. That'll help me/us see how often they're being used and will help eliminate the, and this is your word, "shit". :smile: